qcodes/tests/drivers/test_AimTTİ_PL601P.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from qcodes.instrument_drivers.AimTTi.AimTTi_PL601P_channels import AimTTi
+
+import qcodes.instrument.sims as sims
+visalib = sims.__file__.replace('__init__.py', 'AimTTi_PL601P.yaml@sim')
+
+
+@pytest.fixture(scope='function')
+def driver():
+    driver = AimTTi('AimTTi', address='GPIB::1::INSTR', visalib=visalib)
+
+    yield driver
+    driver.close()
+
+
+@pytest.fixture(scope='function')
+def channels(driver):
+    ch_names = {'ch1'}
+    assert ch_names == set(list(driver.submodules.keys()))
+
+    yield tuple(getattr(driver, ch_name)
+                for ch_name in ch_names)
```

Comment:
Is this fixture ever used?

Reply:
Currently not. I will put a small test to use the fixture for the time being, as I want to keep it for future. I will figure out pytest-sims issue and complete the tests eventually.

Reply:
Ok I removed it, did not make any sense to have it there.